### ISSUE TITLE
Error on Search Providers Page

### ISSUE THREAD

**Issue by jj5000** [SickChill/SickChill#4703]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit:
OS: Windows 10
Browser: Chrome
What you did: Opened the Search Providers Page
What happened: Mako Error
What you expected: To see the list of providers and make changes
Logs:
```
File C:/SickRage/gui/slick/views/config_providers.mako:183, in render_pages:
${('', 'checked="checked"')[curNewznabProvider.enable_daily]}

TypeError: tuple indices must be integers, not unicode
```

**Comment by ravensorb** [comment]
Getting the same issue after updated to the latest build

**Comment by jj5000** [comment]
Rolled back to 39c5dc6 and it works so it is one of the recent changes that has caused the issue

**Comment by ravensorb** [comment]
is this a duplicate of #4721

**Comment by jj5000** [comment]
Yes.  Can confirm the fix from @SudoHenk from #4721 works
